pending/exports-and-boilerplate.pod

```diff
@@ -0,0 +1,116 @@
+=encoding utf8
+
+=head1 Reducing boilerplate and managing exports in Dancer2
+
+Dancer makes it easy to start small and grow fast. Today we will be looking at a techique to help manage that growth.
+
+A typical mature Dancer2 app will span multiple packages and each package will make use of Dancer's plugins and other modules available on CPAN.
+
+  use Scalar::Util qw< blessed >;
+  use List::Util qw< first all none sum >;
+  use Dancer2 appname => MyApp;
+  use Dancer2::Plugin::DBIC qw< schema resultset >;
+  use Dancer2::Plugin::Redis;
+
+After the first couple of files, the copy-pasting becomes tedious and if you later change your mind about what to include, you leave yourself open to runtime errors when you try to call a function you never imported into the package namespace.
+
+=head2 What Dancer2 "exports"
+
+Dancer2 "exports" almost eighty functions into the symbol table by default. That's a lot! You can get a complete list of them from the manual or from the symbol table itself with this one-liner:
+
+  perl -le 'use Dancer2; print for grep { main->can($_) && s/^\*main:://} map {$::{$_}} sort keys %main::;'
+
+(NB: If the concept of exporting and symbol tables in Perl is new to you, read C<perldoc -f use>. The documentation for L<Exporter> may also help you as we go along, but bear in mind that not all Perl modules - including Dancer2 - make use of Exporter.)
+
+For some applications this might present a problem: for instance, if you want to use L<List::Util/any> in a package where Dancer2 has been C<use>d, you cannot import it. Instead, you can qualify the C<any> from L<List::Util>:
+
+  any '/page/:id' => sub {
+    my $id = param('id');
+    forward '/401' if List::Util::any { $_ eq $id } @blocked_resources;
+    pass;
+  };
```

Comment:
I find this code a bit confusing. We generally don't recommend using `forward` or `pass` but this code uses both, and I don't fully understand its purpose. Any chance of changing the forward to a `send_error` and replacing `pass` with `...` or a template render or something?